tests/transforms/test_convert_rois.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+import json
+import os
+import random
+
+from ophys_etl.transforms.convert_rois import (BinarizerAndROICreator,
+                                               BinarizeAndCreationException)
+
+
+@pytest.mark.parametrize("s2p_stat_fixture, ophys_movie_fixture, "
+                         "motion_correction_fixture, seed, binary_quantile, "
+                         "abs_threshold, x_fail, file_remove, x_error",
+                         [({}, {}, {}, 0, None, None,
+                           False, None, None),
+                          ({}, {}, {}, 0, 0.2, None,
+                           False, None, None),
+                          ({}, {}, {}, 0, -1, None,
+                           True, None, BinarizeAndCreationException),
+                          ({}, {}, {}, 0, None, -1,
+                           True, None, BinarizeAndCreationException),
+                          ({}, {}, {}, 0, None, None,
+                           True, 'suite2p_stat_path',
+                           BinarizeAndCreationException),
+                          ({}, {}, {}, 0, None, None,
+                           True, 'motion_corrected_video',
+                           BinarizeAndCreationException)
+                          ],
+                         indirect=["s2p_stat_fixture",
+                                   "ophys_movie_fixture",
+                                   "motion_correction_fixture"])
+def test_binarize_and_convert_rois(s2p_stat_fixture, ophys_movie_fixture,
```

Comment:
There's a lot going on here that would be clearer if it was broken out into separate tests. Let's discuss offline.

Reply:
I separated into tests for the schema validation tests and actual run tests